module/API/Controller.h

```diff
@@ -56,10 +56,25 @@ class Controller {
    */
   void stopArmMotor();
 
+  /**
+   * @brief 右タイヤのPWMを取得する
+   * @return 右タイヤのPWM
+   */
+  static double getRightPwm();
```

Comment:
```suggestion
  static double getRightMotorPwm();
```
他の変更に合わせるなら、ここはMotorの方がいいかな。